deployment/manifests/default.pp

```diff
@@ -55,5 +55,4 @@
 include refinery::neo4j
 include refinery::postgresql
 include refinery::python
-include refinery::selenium
```

Comment:
I think this should be brought back if we're going to keep selenium around and ween ourselves off of it.